cucumber-rails.gemspec

```diff
@@ -1,6 +1,5 @@
 # frozen_string_literal: true
 
-require 'rake/file_list'
```

Comment:
I've based this PR on #496 because this `rake/file_list` require was breaking the build in GitHub Actions.